docs/config.rst

```diff
@@ -308,3 +308,9 @@ Sets authentication provider
   - GitHub `flower.views.auth.GithubLoginHandler`
 
 See `Authentication` for usage examples
+
+
+purge_offline_workers
+~~~~~~~~~~~~~
```

Comment:
There should be a bit more `~` in order to fit the `purge_offline_workers` string length.

```suggestion
~~~~~~~~~~~~~~~~~~~~~
```